common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -3,17 +3,19 @@
 """
 import logging
 
-from xblock.core import XBlock
-from xblock.fields import Scope, String, List
-from xblock.fragment import Fragment
 from xmodule.studio_editable import StudioEditableModule
 
+from xblock.fields import Scope, String, List, Boolean
+from xblock.fragment import Fragment
+from xmodule.library_content_module import XBlock
```

Comment:
This doesn't make sense to me. Shouldn't this line just be `from xblock.core import XBlock` ?